libs/components/datetime/src/lib/modules/datepicker/datepicker-calendar.component.spec.ts

```diff
@@ -444,17 +446,15 @@ describe('datepicker calendar', () => {
       });
 
       componentFixture.debugElement
-        .query(By.css('.sky-datepicker-calendar-inner'))
+        .query(By.css('.sky-datepicker-calendar-table-container'))
         .triggerEventHandler('keydown', eventObj);
       componentFixture.detectChanges();
     }
-    it('should do nothing on shift or alt', () => {
```

Comment:
This test was here specifically to ensure that "Shift" + "Enter" or "Alt" + "Enter" doesn't trigger things. Is there a reason we are removing that?

Reply:
I seem to see a regression here when manually testing. "Shift" + "Enter" is definitely selecting dates.

Reply:
Suppressing the Alt key was breaking screen reader compatibility because the Alt key is how those users send keyboard events outside the sandbox. I can investigate not allowing Shift and Alt specifically for selecting dates.

Reply:
Removed to reexamine in future work